db/schema.rb

```diff
@@ -40,6 +40,14 @@
     t.index ["user_id"], name: "index_announcements_on_user_id"
   end
 
+  create_table "contents", force: :cascade do |t|
+    t.text "body"
```

Comment:
Nit: The schema needs committed again after adding the null false constraints.

Reply:
Good catch :muscle: